use/faq.rst

```diff
@@ -317,7 +317,7 @@ Then, follow the :ref:`install guide <use-install>`. In short, just run this::
     sudo python3 -m pip install limnoria --upgrade
 
 
-.. _blocks: http://freenode.net/faq.shtml#blockingmessages
+.. _blocks: https://libera.chat/guides/usermodes#main
```

Comment:
I think this may have been a broken link even before the freenode -> Libera.Chat move happened and sadly Libera.Chat didn't have a similar FAQ, but as the context is from umode +R, I thought that might be the best equivalent link.